.github/workflows/publish.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Dart
         uses: dart-lang/setup-dart@v1
         with:
-          sdk: 2.12.0
+          sdk: 2.18.6
```

Comment:
why not `stable`? We use it only for publishing anyway.

Reply:
I want to match the test. `stable` includes major changes